windows-terminal/README.md

```diff
@@ -0,0 +1,184 @@
+# Windows terminal
+
+Windows Terminal is the great new one. It includes a lot of cool features like tabs, multiple CLI, and settings to make it nice :smile:
+
+But, you may want to turn it into WOW.
+
+## Overview
+
+Here, we will describe all the steps to make your terminal look astonishingly WOW so that your coworkers will be jealous how much you can `Shell Like a Badass` :sunglasses:
+
+![WT Like a badass](images/wt-like-badass.png)
+
+> :memo: NOTE :memo:
+>
+> It could be installed separately, but since we are lazy enough, we will be using [Scoop](../scoop/README.md).
+
+## Installation Instructions
+
+> :warning: WARNING :warning:
+>
+> We will assume that your current terminal will be at the root of the repo.
+
+### Install Scoop
+
+First, if it's not already installed (what are you waiting for!), you will need to install scoop using:
+
+```powershell
+./scoop/scoop.base.install.ps1
+```
+
+#### Adding Buckets
+
+You will also need specific known buckets for this to work:
+
+```powershell
+scoop bucket add extras
+scoop bucket add nerd-fonts
+```
+
+### Install all required applications
+
+Because you are using scoop, the installation of ALL the apps will be easy. Following these instructions will install:
+
+- Windows-Terminal : The new terminal in the Block
+- PowerShell 7 : The successor to PowerShell 5 shell
+- Posh-Git : Git auto completion in PowerShell
+- Oh-My-Posh3 : Prompt theme engine
+- Sudo : To force elevated privilege
+- Cascadia-Code fonts
+
+```powershell
+# you should always update scoop before working with it
+scoop update
+scoop install windows-terminal pwsh posh-git oh-my-posh3 sudo
+sudo scoop install Cascadia-Code CascadiaCode-NF CascadiaCode-NF-Mono
+```
+
+> :memo: NOTE :memo:
+>
+> Oh-My-Posh3 support other shell like Bash, Zsh, ... but, it will not be explain in this post. You can get more information on [installation documentation](https://ohmyposh.dev/docs/installation) page of oh-my-posh3.
+
+## Configuration
+
+### Enabling Oh-My-Posh3 and Posh-Git in PowerShell 7
+
+These steps will enable Oh-My-Posh3 and Posh-Git
+
+First, go to `windows-terminal/config` and manually copy the folder "PowerShell" (and its content) into in your `$env:UserProfile/Documents`. 
```

Comment:
- into in your => typo, into you*

I found it a bit confusing, it's a mix of folder and powershell but we manually copy from windows explorer.

Will $env:UserProfile/Documents work on linux? If not, I would be more explicit about powershell or suggest something like :
Open windows explorer and copy windows-terminal/config into %userprofile%/Documents

Reply:
Typo should be fix in new the PR. In that same PR I also added some explanations to reduce the confusion and added a notice that this page was intented to windows... you can tell me what you think in the PR #4 ;)